### PR TITLE
{chem}[system/system] Chemaxon-Marvin v23.9

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/f4b7088f268896643c5128b799e55134 for a full test report.

**Comment by SIB-software** [comment]
Test report by @sib-software
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vitbuild - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/SIB-software/7a722771ff386c557ec81f4599b26029 for a full test report.

File: easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+sanity_check_commands = ["molconvert"]
+
+# You need a key/license file to fully activate Chemaxon Marvin
+#
+# COMMERCIAL
+# For commercial use please contact sales@chemaxon.com
+# NON-COMMERCIAL
+# For personal   use get a key to be renewed every 2 months at https://chemaxon.com/products/marvin/download
+# For academic   use get a key for research or educational purposes at https://chemaxon.com/academic-license
```
Comment (verdurin) [inline easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb:68]: This looks like an old URL - gives `404` for me.

File: easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+software_license_urls = ['https://www.chemaxon.com/marvin/license.html']
+docurls = [
+    'https://docs.chemaxon.com/display/docs/marvinsketch-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/marvinview-user-s-guide.md',
+    'https://docs.chemaxon.com/display/docs/file-formats.md',
+]
+
+toolchain = SYSTEM
+
+# Download and license at https://chemaxon.com/products/marvin
```
Comment (verdurin) [inline easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb:33]: Not clear that I can use any of their license options for testing, really.

Comment (smoretti) [reply easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb:33]: Maybe better now with updated URLs.

But this tool is like matlab, you may not have access to the source package. That's why for the first eb file of this tool they asked me to upload a test-report. What I did this morning

File: easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb]: Thanks.

File: easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-23.9.eb:33]: Sure. If we can only provide limited testing, that's no reason not to include it.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @SIB-software!

**Pull request merged**